libtiledbsoma/src/utils/fastercsx.h

```diff
@@ -198,8 +198,13 @@ void count_rows_(
                         auto row = Ai_view[n];
                         if ((row < 0) ||
                             (static_cast<std::make_unsigned_t<COO_IDX>>(row) >=
-                             n_row)) [[unlikely]]
-                            throw std::out_of_range("Coordinate out of range.");
+                             n_row)) [[unlikely]] {
+                            throw std::out_of_range(std::format(
```

Comment:
Suggest changing message to "First dimension coordinate {} out of range {}."

In other words, indicate if it is a coordinate in I or J vector which it out of range.